File: agent/bindings/kotlin/lib/src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/SaveDialogOptionsParams.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class SaveDialogOptionsParams(
  val defaultUri: String? = null,
  val saveLabel: String? = null,
  val filters: Map<String, List<String>>? = null,
  val title: String? = null,
)

File: agent/bindings/csharp/SaveDialogOptionsParams.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class SaveDialogOptionsParams
  {
    [JsonProperty(PropertyName = "defaultUri")]
    public string DefaultUri { get; set; }
    [JsonProperty(PropertyName = "saveLabel")]
    public string SaveLabel { get; set; }
    [JsonProperty(PropertyName = "filters")]
    public Dictionary<string, string[]> Filters { get; set; }
    [JsonProperty(PropertyName = "title")]
    public string Title { get; set; }
  }
}
